src/components/TimelineNav/TimelineNav.tsx

```diff
@@ -428,11 +444,17 @@ export const TimelineNav = ({
         <Button
           className={clsx(styles['timeline-nav__back'])}
           onClick={onClick}
+          onFocus={onFocus}
           ref={backRef}
           status="neutral"
           variant="link"
         >
-          <Icon name="arrow-back" purpose="informative" title="back" />
+          <Icon
+            id="back"
```

Comment:
This is a relic from my first attempt at fixing this issue. It has nothing to do with the work in this ticket, I just forgot to take it out.

Reply:
Can you take it out?